TESTS/integration/COMMON/common_defines_fs_test.h

```diff
@@ -50,15 +29,14 @@
 #elif COMPONENT_SD
 #define TEST_BLOCK_DEVICE_TYPE "SD"
 #define TEST_USE_FILESYSTEM FS_FAT
-#elif COMPONENT_FLASHIAP
-#define TEST_BLOCK_DEVICE_TYPE "FLASHIAP"
```

Comment:
Why removing `FLASHIAP` ? is this cause trouble for many targets ?
In some case we still rely on this, like FastModels

Reply:
Can we leave the FLASHIAP there ?